src/protocol.cpp

```diff
@@ -176,6 +180,8 @@ std::string CInv::GetCommand() const
     case MSG_BLOCK:          return cmd.append(NetMsgType::BLOCK);
     case MSG_FILTERED_BLOCK: return cmd.append(NetMsgType::MERKLEBLOCK);
     case MSG_CMPCT_BLOCK:    return cmd.append(NetMsgType::CMPCTBLOCK);
+    case MSG_MWEB_HEADER:    return cmd.append(NetMsgType::MWEBHEADER);
+    case MSG_MWEB_LEAFSET:    return cmd.append(NetMsgType::MWEBLEAFSET);
```

Comment:
There's an extra space ruining the alignment.

Reply:
I don't know how my OCD missed that.